chanzuckerberg/edu-design-system
# Release v1.0.0 🎉

### Summary:
see CHANGELOG! main change in this release is updated color token hex values

### Test Plan:
- build passes
- check chromatic
